spec/models/chargeback_vm_spec.rb

```diff
@@ -383,15 +383,18 @@ def pluck_rollup(metric_rollup_records)
 
         let(:development_vm) { FactoryBot.create(:vm_vmware, :created_on => month_beginning) }
         let(:other_vm)       { FactoryBot.create(:vm_vmware, :created_on => month_beginning) }
+        let(:another_vm)     { FactoryBot.create(:vm_vmware, :created_on => month_beginning) }
 
         before do
           environment_category = Classification.find_by(:description => "Environment")
           tag_development = FactoryBot.create(:classification, :name => "dev", :description => "Development", :parent_id => environment_category.id)
 
           development_vm.tag_with(tag_development.tag.name, :ns => '*')
 
-          metric_rollup_params.delete(:tag_names)
+          metric_rollup_params[:tag_names] = ""
```

Comment:
not sure if this is what we want. I think we want no tags (rather than having a blank tag)

Reply:
Actually, I think we need to test with both no tags and empty string because I'm seeing both cases in the data I've collected -
```
2.5.3 :048 > MetricRollup.where.not(:tag_names => "").count
   (1839.1ms)  SELECT COUNT(*) FROM "metric_rollups" WHERE "metric_rollups"."tag_names" != $1  [["tag_names", ""]]
 => 77780
2.5.3 :049 > MetricRollup.where.not(:tag_names => nil).count
   (273.0ms)  SELECT COUNT(*) FROM "metric_rollups" WHERE "metric_rollups"."tag_names" IS NOT NULL
 => 155262
```
So I'll add that test back, thanks!